lib/components/Keyboard/KeyboardInput/KeyboardAccessoryView.js

```diff
@@ -110,7 +110,18 @@ class KeyboardAccessoryView extends Component {
      *
      * default: false
      */
-    allowHitsOutsideBounds: PropTypes.bool
+    allowHitsOutsideBounds: PropTypes.bool,
+
+    /**
+     * iOS only.
+     * Set false to allow the KeyboardAccessoryView to reach to the bottom of the screen,
```

Comment:
The language is a bit cumbersome, I don't think you need to explain it in depth.
We have these two for other components:
1. "Whether or not to handle SafeArea"
2. "Use safe area, in case action bar attached to the bottom (default: true)"
Maybe something like:
"Whether or not to handle SafeArea

default: true"